### ISSUE TITLE
epic: stuck on EULA (End User License Agreement) -> changed locator

### ISSUE THREAD

**Issue by darkero27** [vogler/free-games-claimer#371]
how do i resolve this?
![image](https://github.com/user-attachments/assets/e12640ff-70e3-4971-a4eb-19866b343329)

**Comment by RonSijm** [comment]
I have the same, there might be some kinda random popup you have to click on (like a new agreement)

run `npx cross-env SHOW=1 node epic-games` instead, and see what the browser is doing. You probably have to click something

**Comment by darkero27** [comment]
> I have the same, there might be some kinda random popup you have to click on (like a new agreement)
> 
> run `npx cross-env SHOW=1 node epic-games` instead, and see what the browser is doing. You probably have to click something
![image](https://github.com/user-attachments/assets/e6b27378-ba78-436d-af2d-1e4117856b5d)
yeah, the happend is eula thingy

**Comment by vogler** [comment]
Mh, that should be accepted automatically:
https://github.com/vogler/free-games-claimer/blob/eafb2316f71d0cc548c4263be74c1818114ffc5b/epic-games.js#L217-L222
I guess the locator `input#agree` changed. Sadly this only happens once per account, so it's not easy to recreate and find out what it needs to be changed to (except by creating a new account).

**Comment by vogler** [comment]
It now reacts to the text, dumps the page's HTML and asks the user to report it here.

**Comment by darkero27** [comment]
> It now reacts to the text, dumps the page's HTML and asks the user to report it here.

![image](https://github.com/user-attachments/assets/93700454-94e7-4077-8318-6f6e515d906a)

update UI location on claim, it happend on new game

**Comment by nicklutt** [comment]
> here.
> 

Here it's the same screen, and then a timeout.

**Comment by vogler** [comment]
@darkero27 Sep. issue #374, just closed.

**Comment by vogler** [comment]
This issue (EULA) should not be closed since we're still missing the updated locator to automatically accept it.

**Comment by darkero27** [comment]
> This issue (EULA) should not be closed since we're still missing the updated locator to automatically accept it.

Owh sorry, I'm not used to using github

**Comment by RonSijm** [comment]
> I guess the locator `input#agree` changed. Sadly this only happens once per account, so it's not easy to recreate and find out what it needs to be changed to (except by creating a new account).

I don't think it happens exactly "once per account" though - I've already had this account for long time, and also used this free-games-claimer for a while.

I think it might be happening either when the EULA has changed, or just 'sometimes random' - I guess

**Comment by vogler** [comment]
Haven't compared the text, but yea, I assume also on changes, don't think random. At least for me I think I only saw this 2 or 3 times pop up.

**Comment by IAteYourCookies** [comment]
I've had the locator timeout pop up almost every day for the past 2 weeks now.
![image](https://github.com/user-attachments/assets/837dc00e-8104-4e45-a110-10c943e65e75)

**Comment by vogler** [comment]
@IAteYourCookies What line is this? It's not part of the EULA code.

Did someone get the HTML or a screenshot of the DOM when the EULA was shown?
